### DISCUSSION TITLE
Hen House Upgrade

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#2615] [Ideas]
![image](https://github.com/sunflower-land/sunflower-land/assets/133494217/e26d5dd8-051b-4016-878b-74c860ad6218)
Two chicken coops, taking up a lot of space. I propose that instead of building a second coop. make an improvement from level 1 to level 2, and then possibly to the third level.  By simply increasing the size of the henhouse, for example as in the screenshot 
![image](https://github.com/sunflower-land/sunflower-land/assets/133494217/b08b65cc-804e-4f4a-a780-15fac654cf2a)

**Comment by TheDcol** [comment]
The idea of multiple buildings is indeed to take up space, further incentivising players to expand.
I am not against the idea of upgrading! But that is the reasoning as to why it might not happen

**Reply by Dionis404** [reply]
In the game only the henhouse can be built two times. As far as I remember now that to get access to the construction of the building must have a certain level of expansion. So it makes no sense what you say. To build a building still need to expand the farm.

**Reply by eliasSFL** [reply]
> The idea of multiple buildings is indeed to take up space, further incentivising players to expand.
> I am not against the idea of upgrading! But that is the reasoning as to why it might not happen

@TheDcol something they could do is to lock certain level upgrades to number of expansions, rather than having a second building.

**Reply by Dionis404** [reply]
![image](https://github.com/sunflower-land/sunflower-land/assets/133494217/d6c92c7c-303e-4308-adc3-ba7e010b7cdb)
I took a screenshot from the test network.

**Reply by TheDcol** [reply]
I am just communicating what the devs have said before
